classes/Customization.php

```diff
@@ -277,6 +277,8 @@ public static function retrieveQuantitiesFromIds($idsCustomizations)
     }
 
     /**
+     * @deprecated 8.0.0 This was needed to validate order canceling
+     *
```

Comment:
Could you deprecate the method using the method explained in devdocs (https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations) ?

Reply:
Resolved in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/3d20bbc2854ff0063ff10280afdcb0f1464f5d3b